plugins/Login/Login.php

```diff
@@ -65,7 +65,7 @@ private function shouldHandleRememberMe()
     {
         $module = Common::getRequestVar('module', false);
```

Comment:
Not quite sure I understand `shouldHandleRememberMe`. Shouldn't remember me be always handled? there could be also maybe a login form when opening multi sites etc?

Reply:
In the sessions PR we had a discussion about whether it would be possible for someone to add form_rememberme to a URL and have it extend the session for the user, and this was a way to mitigate it. Basically, if we're detecting a POST by the login form, then we check form_rememberme & change the session time (because if it's not set, then we want it to end with the browser session).